server.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const app = express();
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 3000;
```

Comment:
The client-side is served on port 3000 and proxies all requests to 5000, why did you change the server port?

Reply:
Do you mind changing it back, so we can close the PR